src/_guides/language/spec.md

```diff
@@ -1,73 +1,33 @@
 ---
 layout: default
 title: "Dart Language Specification"
-description: "The Dart language specification and proposed changes."
+description: "The Dart language specification."
```

Comment:
Nits:
- The "layout: default" can be dropped.
- No need for quotes around the `title`/`description` values.